upgrading to 3.0.0-alpha.3 breaks compilation

On my current project this:

```
-    "ra-core": "^3.0.0-alpha.2",
-    "ra-data-graphql-simple": "^3.0.0-alpha.1",
+    "ra-core": "^3.0.0-alpha.3",
+    "ra-data-graphql-simple": "^3.0.0-alpha.3",
-    "react-admin": "^3.0.0-alpha.2",
+    "react-admin": "^3.0.0-alpha.3",
```
results in this error:
```
Failed to compile.

./node_modules/ra-ui-materialui/esm/layout/DeviceTestWrapper.js
Module not found: Can't resolve 'css-mediaquery' in '/myproject/node_modules/ra-ui-materialui/esm/layout'
```

--- a/packages/ra-ui-materialui/package.json
+++ b/packages/ra-ui-materialui/package.json
@@ -28,7 +28,6 @@
     "devDependencies": {
         "@testing-library/react": "^8.0.7",
         "cross-env": "^5.2.0",
-        "css-mediaquery": "^0.1.2",
         "enzyme": "~3.9.0",
         "enzyme-adapter-react-16": "~1.12.1",
         "file-api": "~0.10.4",
@@ -49,6 +48,7 @@
         "autosuggest-highlight": "^3.1.1",
         "classnames": "~2.2.5",
         "connected-react-router": "^6.4.0",
+        "css-mediaquery": "^0.1.2",
         "final-form": "^4.18.5",
         "final-form-arrays": "^3.0.1",
         "inflection": "~1.12.0",

